ports/esp32/esp32_rmt.c

```diff
@@ -103,8 +121,14 @@ STATIC mp_obj_t esp32_rmt_make_new(const mp_obj_type_t *type, size_t n_args, siz
 STATIC void esp32_rmt_print(const mp_print_t *print, mp_obj_t self_in, mp_print_kind_t kind) {
     esp32_rmt_obj_t *self = MP_OBJ_TO_PTR(self_in);
     if (self->pin != -1) {
-        mp_printf(print, "RMT(channel=%u, pin=%u, source_freq=%u, clock_div=%u)",
+        mp_printf(print, "RMT(channel=%u, pin=%u, source_freq=%u, clock_div=%u",
             self->channel_id, self->pin, APB_CLK_FREQ, self->clock_div);
+        if (self->carrier_en) {
```

Comment:
for this test use `self->carrier_freq > 0`

Reply:
> @JonRob812 is there a reason you didn't include `carrier_level` as an option? Maybe it's not needed after all, since all the IDF examples set it to 1. Anyway, it can be easily added if needed (maybe later on?) by a new kwarg.


@dpgeorge I see same thing looking at IDF docs. I would love to add it but it seems wrong to write code without fully understanding the context in which it operates.  Is it a simple invert? if so, it seems like this can be accomplished this with the `start=1` keyword in `write_pulses()`? 

Long story short, I lack of the full understanding that variable, `idle_output_en`, and `idle_level`.